modules/apps/notification/notification-rest-api/src/main/java/com/liferay/notification/rest/dto/v1_0/util/NotificationUtil.java

```diff
@@ -172,6 +172,8 @@ public static NotificationTemplate toNotificationTemplate(
 		serviceBuilderNotificationTemplate.setSubjectMap(
 			LocalizedMapUtil.getLocalizedMap(
 				notificationTemplate.getSubject()));
+		serviceBuilderNotificationTemplate.setSystem(
```

Comment:
Move NotificationUtil.java from `notification-rest-api` to `notification-rest-impl`